src/wallet/rpcwallet.cpp

```diff
@@ -3484,37 +3490,38 @@ UniValue spendzerocoin(const UniValue& params, bool fHelp)
     if(sporkManager.IsSporkActive(SPORK_16_ZEROCOIN_MAINTENANCE_MODE))
         throw JSONRPCError(RPC_WALLET_ERROR, "zPIV is currently disabled due to maintenance.");
 
-    EnsureWalletIsUnlocked();
```

Comment:
should probably keep this check, as a locked wallet will ultimately fail `CWallet::SpendZerocoin()` anyways.